### PR TITLE
#39 - Adds Base Configuration Changes for ESLint

### PR DESCRIPTION
**Pull request by GoodGuyGregory**
@leekahung This expands on what we discussed in the meeting this week. 

It should add base configurations. I recommend we go line by line with this one locally, if all of your changes are up to date in Development, 

I should have some time Friday if you're available on Discord

### CODE REVIEW

File: .prettierignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+./docs
+./node_modules
+./src/index.html
```
Comment (leekahung) [inline .prettierignore:3]: I don't think Line 3 is necessary. The base index.html is located on the root directory: ./

File: src/App.jsx [hunk 2]
```diff
@@ -1,37 +1,30 @@
...
+import CrossPodWrite from './components/Form/CrossPodWrite';
 
-const AppHeader = () => {
-  return (
-    <header>
-      <h2>Getting Started with PASS</h2>
-    </header>
-  );
-};
+const AppHeader = () => {return (
```
Comment (leekahung) [inline src/App.jsx:8]: Can review arrow function block statements for React in Friday's discussion, but essentially, components can be made without explicit returns if it's only return JSX Elements: https://stackoverflow.com/questions/53203257/making-components-with-curly-brackets-versus-parenthesis-when-using-functional-c

There's an example of what's shown in StackOverflow in Development's updated AppHeader without the explicit return.

File: src/components/Form/CrossPodWrite.jsx [hunk 3]
```diff
@@ -1,95 +1,77 @@
-import { useState } from "react";
-import { useSession } from "@inrupt/solid-ui-react";
-import { useField } from "../../hooks";
-import StatusNotification from "./StatusNotification";
-import DocumentSelection from "./DocumentSelection";
+import { useState, React } from 'react';
```
Comment (leekahung) [inline src/components/Form/CrossPodWrite.jsx:1]: Should be: import React, { useState } from 'react';

File: src/components/Form/DocumentSelection.jsx [hunk 4]
```diff
@@ -1,15 +1,15 @@
...
-        {docTypes.map((doc, index) => {
-          return <option key={index}>{doc}</option>;
-        })}
-      </select>
-    </>
-  );
+	return (
+		<select id={htmlId} name="document">
+			{docTypes.map((doc, index) => {
+				return <option key={index}>{doc}</option>;
```
Comment (leekahung) [inline src/components/Form/DocumentSelection.jsx:8]: Found a workaround for key prop. Decided to just use the first word of multiworded docType, which can be obtained using: doc.split(' ')[0]. Can be discussed on Friday.

Check out Development's DocumentSelection.jsx for the workaround.

**Review by leekahung — CHANGES_REQUESTED** [review]
These feedback will likely be discussed on our Friday discussion. Don't see much of a problem with lint rules being presented here. It's very similar to what I've been exploring. Just need to hash things out by line as mentioned on Discord on what rules to add or drop. After that, I think we're good to merge based on the changes to ESLint/Prettier config files and updates from reformatting using Prettier.

File: .prettierignore (hunk 1 above)
Comment (GoodGuyGregory) [reply .prettierignore:3]: removing now

**Pull request closed without merging**